### DISCUSSION TITLE
Running CFS for rtems errors

### DISCUSSION THREAD

**Discussion by polaraf** [nasa/cFS#498] [Q&A]
sample_app.obj dl_info fails to load symbols  even after compiling it statically 

unresolved symbol: CFE_ES_PerfLogAdd
unresolved symbol: CFE_SB_ReceiveBuffer
unresolved symbol: CFE_EVS_SendEvent

we compiled cfs for rtems 4.11 and trying to run it on leon3 processor 

any idea why it fails to load symbols ?

**Comment by ghost** [comment]
Where is this error being reported? Is it when the cFE core tries to load sample_app.obj?

Are you linking the cFE core to the RTEMS image? ( like in the pc-rtems platform)
You can check the images for symbols:

```
$ nm (or sparc-rtems4.11-nm) core-cpu1.exe | sort > map.txt
```

I believe you can also list the undefined symbols in an object

```
$ nm -u sample_app.obj
```

The Code (Text) symbols should be in the core image. They should satisfy the undefined symbols in sample_app.obj

Since we are considering deprecating RTEMS 4.11 support in the cFS, can I ask why you use RTEMS 4.11 and not 5.1?

Alan
